src/PKSim.Core/Snapshots/Mappers/ExtendedPropertyMapper.cs

```diff
@@ -17,12 +17,10 @@ public override Task<SnapshotExtendedProperty> MapToSnapshot(ModelExtendedProper
          {
             snapshot.ListOfValues = extendedProperty.ListOfValuesAsObjects.Any() ? extendedProperty.ListOfValuesAsObjects.ToList() : null;
             snapshot.Value = extendedProperty.ValueAsObject;
-            snapshot.Type = extendedProperty.Type;
             snapshot.Description = SnapshotValueFor(extendedProperty.Description);
             snapshot.Name = extendedProperty.Name;
-            snapshot.FullName = extendedProperty.FullName;
-            snapshot.DisplayName = extendedProperty.DisplayName;
-            snapshot.ReadOnly = extendedProperty.ReadOnly;
+            snapshot.FullName = string.Equals(extendedProperty.FullName, extendedProperty.Name) ? null : extendedProperty.FullName;
```

Comment:
Here you can use the SnapshotValueFor which takes a second parameter( default)

Reply:
Actually, I think I can leave this with SnapshotValueFor without second parameter because FullName can be null

Reply:
T must be non-nullable to use this default value version right?

Reply:
there is an overload for string

Reply:
right but it does not take default parameter :)

Reply:
Yeah, I think it's ok to use the standard one for string. If FullName was set, then it should be preserved, otherwise it's blank